packages/react/src/ImageSlider/ImageSlider.tsx

```diff
@@ -69,19 +86,64 @@ export const ImageSliderRoot = forwardRef(
     useEffect(() => {
       const observer = new IntersectionObserver(inView, observerOptions)
 
+      const handleKeyDown = (event: KeyboardEvent) => {
+        switch (event.key) {
+          case 'ArrowLeft':
+            goToPreviousSlide()
+            break
+          case 'ArrowRight':
+            goToNextSlide()
+            break
+        }
+      }
```

Comment:
Can we use a lookup object instead of a switch?

```suggestion
      const keyActionMap = {
        ArrowLeft: goToPreviousImage,
        ArrowRight: goToNextImage,
      }

      const handleKeyDown = (event: KeyboardEvent) => {
        const action = keyActionMap[event.key as keyof typeof keyActionMap]
        action?.()
      }
```

Reply:
This doesn't improve readability.

Reply:
Well, it’s not that difficult to read (it appears that the cast wasn’t needed, so I removed it), while it does improve maintainability. And the actions aren’t undefined; maybe you tried after renaming the functions? I updated these as well.